### DISCUSSION TITLE
Создание внешней формы

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#351] [ОШБК!] [answered]
Добрый день!

Прошу помочь. Создал внешнюю форму для сбора информации от пользователей. Делаю все как в описании https://ru.docs.totum.online/forms, но созданная форма не загружается в вэбе, а просто крутится загрузка бесконечно. В чем может быть проблема?

**Comment by erpic-ru** [comment, marked as answer]
Попробуйте обновиться до последней версии

**Reply by ttm-alexeypolunin** [reply]
Было несколько версий с этой ошибкой - ее поправили неделю назад. Надо обновить: https://ru.docs.totum.online/update

**Reply by Spyker086** [reply]
Ок, Спасибо!

**Reply by Spyker086** [reply]
После обновления форма стала загружаться. На моей машине (разарабатываю форму) теперь такая штука: несколько раз форма загружается нормально, но потом перестают загружаться поля формы (пустая форма). Коллегам ссылку передаю такой проблемы нет. Как можно профиксить?

Помогла очистка куков

**Reply by Spyker086** [reply]
Еще один момент выявлен: заполняем форму, но в любом из полей где есть проверка на ввод или поле обязательное, но его оставили пустым. Нажимаем сохранить, форма выдает ошибку. Возвращаемся исправляем данные и кнопка Сохранить перестает работать. Обновляем форму, вводим все корректно - кнопка Сохранить данные в таблицу записывает. В чем может быть проблема?

**Reply by ttm-alexeypolunin** [reply]
Напишите мне в телеграм @alexeypolunin, через пару часов посмотрим у вас тк у нас не повторяется

**Reply by ttm-alexeypolunin** [reply]
Хотя не, я потыкал основательно и смог повторить оба на быстрой форме. Поправим, напишу сюда или вам уже в телеграм если вы мне туда напишете.
